pathz/pathz.proto

```diff
@@ -72,60 +67,32 @@ service Pathz {
   //     in the new policy and validates that the new policy "works".
```

Comment:
i think step 3 should be sending 'Probe()' messages and getting replies, not necessarily testing the policy.
The policy may not be 'live' right?

Reply:
You are right - now the policy is in a sandbox until the Finalize message, so there is no other way to check it out than the Probe() RPC.